src/widgets/survey/model/services/tests/testHelpers.ts

```diff
@@ -216,6 +216,7 @@ export const getFlowProgressMock = (): StoreProgress => {
           executionGroupKey: 'mock-flow-group-key-1',
           pipelineActivityOrder: 0,
           totalActivitiesInPipeline: 2,
+          entityName: 'mock-entity-name',
```

Comment:
Change it to empty string, please. The related BL doesn't use this field. And if even so, it'd be easy to catch.